webpack.config.js

```diff
@@ -183,7 +183,7 @@ const client = {
       "@": path.join(__dirname, "src")
     }
   },
-  target: "web",
+  target: ["web", "es5"],
```

Comment:
why do we need `es5` here?

Reply:
This is necessary for IE11 compatibility. Without this, Webpack uses es6 arrow functions when compiling JavaScript bundles.

Reply:
Ah interesting, did not realise that. 😀